### ISSUE TITLE
virtualbox-vm build fails on 1.5.1

### ISSUE THREAD

**Issue by onematchfox** [hashicorp/packer#8575]
### Overview of the Issue

Subsequent to upgrade to Packer 1.5.1, Virtualbox VM build now fails during the Prepare step.

#### Reproduction Steps

See sample build file below.

### Packer version

1.5.1

### Simplified Packer Buildfile

```
{
  "builders": [
    {
      "type": "virtualbox-vm",
      "communicator": "ssh",
      "ssh_username": "x",
      "ssh_private_key_file": "~/.ssh/my_key",
      "shutdown_command": "sudo shutdown -h now",
      "guest_additions_mode": "disable",
      "output_directory": "./builds-vm",
      "vm_name": "existing_vm",
      "format": "ova"
    }
  ]
}
```

### Operating system and Environment details

Windows x64

### Log Fragments and crash.log files

```
2020/01/07 16:23:58 Preparing build: virtualbox-vm
unexpected EOF
2020/01/07 16:23:58 packer.exe plugin: panic: runtime error: invalid memory address or nil pointer dereference
2020/01/07 16:23:58 packer.exe plugin: [signal 0xc0000005 code=0x0 addr=0x0 pc=0x2957164]
2020/01/07 16:23:58 packer.exe plugin:
2020/01/07 16:23:58 packer.exe plugin: goroutine 62 [running]:
2020/01/07 16:23:58 packer.exe plugin: github.com/hashicorp/packer/builder/virtualbox/vm.(*Config).Prepare(0x0, 0xc000109da0, 0x2, 0x2, 0x5e, 0x0, 0x1, 0x2030000, 0x0)
2020/01/07 16:23:58 packer.exe plugin:  /Users/mmarsh/Projects/packer/builder/virtualbox/vm/config.go:50 +0x94
2020/01/07 16:23:58 packer.exe plugin: github.com/hashicorp/packer/builder/virtualbox/vm.(*Builder).Prepare(0xc0001091a0, 0xc000109da0, 0x2, 0x2, 0x2, 0x2, 0x0, 0x0, 0x0, 0x0, ...)
2020/01/07 16:23:58 packer.exe plugin:  /Users/mmarsh/Projects/packer/builder/virtualbox/vm/builder.go:26 +0x57
2020/01/07 16:23:58 packer.exe plugin: github.com/hashicorp/packer/packer/rpc.(*BuilderServer).Prepare(0xc00012dc00, 0xc000109d60, 0xc00012dcc0, 0x0, 0x0)
2020/01/07 16:23:58 packer.exe plugin:  /Users/mmarsh/Projects/packer/packer/rpc/builder.go:97 +0xa7
2020/01/07 16:23:58 packer.exe plugin: reflect.Value.call(0xc000534720, 0xc0001060d0, 0x13, 0x412cb03, 0x4, 0xc0002ddf18, 0x3, 0x3, 0x0, 0x0, ...)
2020/01/07 16:23:58 packer.exe plugin:  /usr/local/go/src/reflect/value.go:460 +0x5fd
2020/01/07 16:23:58 packer.exe plugin: reflect.Value.Call(0xc000534720, 0xc0001060d0, 0x13, 0xc0002ddf18, 0x3, 0x3, 0x0, 0x0, 0x0)
2020/01/07 16:23:58 packer.exe plugin:  /usr/local/go/src/reflect/value.go:321 +0xbb
2020/01/07 16:23:58 packer.exe plugin: net/rpc.(*service).call(0xc00012dc40, 0xc000536280, 0xc0005a1620, 0xc0005a1640, 0xc000234480, 0xc000109ce0, 0x33b0a60, 0xc000109d60, 0x16, 0x33b0aa0, ...)
2020/01/07 16:23:58 packer.exe plugin:  /usr/local/go/src/net/rpc/server.go:377 +0x176
2020/01/07 16:23:58 packer.exe plugin: created by net/rpc.(*Server).ServeCodec
2020/01/07 16:23:58 packer.exe plugin:  /usr/local/go/src/net/rpc/server.go:474 +0x432
2020/01/07 16:23:58 Build 'virtualbox-vm' prepare failure: unexpected EOF
2020/01/07 16:23:58 [INFO] (telemetry) Finalizing.
```

**Comment by azr** [comment]
Hey @onematchfox; thanks for opening; this is something I forgot to fix in #8513 ! I just opened #8576 to fix this. Binaries can be found [here](https://circleci.com/gh/hashicorp/packer/27622#artifacts/containers/0); please tell us if that fixes it for you !

**Comment by onematchfox** [comment]
> Hey @onematchfox; thanks for opening; this is something I forgot to fix in #8513 ! I just opened #8576 to fix this. Binaries can be found [here](https://circleci.com/gh/hashicorp/packer/27622#artifacts/containers/0); please tell us if that fixes it for you !

@azr Wow! Thanks so much for the incredibly quick response and fix. Yes, this change fixed my issue.
